lib/utils.js

```diff
@@ -87,17 +87,17 @@ module.exports.formatNotes = ({ parsed, version, previous }) => {
   let notes = `## Release Notes: from ${previous} to ${version}`
 
   Object.entries(byT1).forEach(([t1, grouped]) => {
-    notes += `\n\n ### ${t1}\n`
+    notes += `\n\n### ${t1}\n`
 
     Object.entries(grouped).forEach(([t2, item]) => {
       // commits without T2 skip the title
-      notes += t2 === 'others' ? '' : `\n * #### ${t2}\n`
+      notes += t2 === 'others' ? '\n* ' : `\n* #### ${t2}\n\t* `
```

Comment:
actually i missed this the last time. using both `h4` (`####`) _and_ list item (`*`) can be a bit overwhelming (visual diff in both size and indentation)

perhaps we can keep it without the list item part and just let the headers fall through. if h4 is too insignificant compared to its parent level header, drop to h5 to see how it looks in rendered markdown

Reply:
ref: https://eqworks.slack.com/archives/CFR7TDMJQ/p1605291586046500?thread_ts=1605284971.040200&cid=CFR7TDMJQ